### PR TITLE
Switch windows gce jobs from dockershim to containerd

### PR DESCRIPTION
**Pull request by marosset**
Signed-off-by: Mark Rossetti <marosset@microsoft.com>

/sig windows
/assign @jsturtevant @pjh 

Also, 2004 is out of service so I removed those jobs.

### CODE REVIEW

**Comment by jsturtevant** [comment]
required for dockerhim removal kubernetes/kubernetes#97252

lgtm but will wait for @pjh or @ibabou

**Comment by pjh** [comment]
LGTM, @ibabou to approve.

File: config/jobs/kubernetes/sig-windows/windows-gce.yaml [hunk 1]
```diff
@@ -103,59 +104,7 @@ periodics:
     fork-per-release-replacements: "--extract=ci/latest -> --extract=ci/latest-{{.Version}}"
     testgrid-dashboards: google-windows, sig-windows-gce
     testgrid-tab-name: gce-windows-2019-master
-    description: Runs tests on a Kubernetes cluster with Windows 2019 nodes on GCE
-
-- name: ci-kubernetes-e2e-windows-gce-20h2
```
Comment (ibabou) [inline config/jobs/kubernetes/sig-windows/windows-gce.yaml:108]: Is there a specific reason to removing the job for gce-windows-20h2-master?

Comment (marosset) [reply config/jobs/kubernetes/sig-windows/windows-gce.yaml:108]: That one was for dockershim.
I left in the 20h2-master-contaienrd job (right below this job definition).

Comment (ibabou) [reply config/jobs/kubernetes/sig-windows/windows-gce.yaml:108]: Oh, my bad. I forgot I had the other separate one for containerd. Makes sense. Thanks Mark for the update! LGTM

**Review by ibabou — APPROVED** [review]

**Comment by mauriciopoppe** [comment]
The preset is adding a few vars, should we remove the preset and change the kubernetes kube-up scripts instead? https://github.com/kubernetes/kubernetes/blob/c75d254beb662ea73736d2f123d0382b7f91dfbd/cluster/gce/config-default.sh?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L102

**Comment by mauriciopoppe** [comment]
the default value in kube-up was changed instead in https://github.com/kubernetes/kubernetes/pull/107189

**Pull request merged**